### ISSUE TITLE
error while generating report

### ISSUE THREAD

**Issue by tuxtechno** [TheGroundZero/openvasreporting#31]
Hi,

I am getting below error while executing the script, can anyone guide please

```bash
sudo python3 -m openvasreporting -i openvas.xml -f xlsx

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/yogesh/openvasreporting/openvasreporting/__main__.py", line 19, in <module>
    main()
  File "/home/yogesh/openvasreporting/openvasreporting/openvasreporting.py", line 70, in main
    config = Config(args.input_files, 
  File "/home/yogesh/openvasreporting/openvasreporting/libs/config.py", line 86, in __init__
    raise ValueError("Invalid value for level parameter, \
ValueError: Invalid value for level parameter,                 must be one of: c[ritical], h[igh], m[edium], l[low], n[one]
```

**Comment by TheGroundZero** [comment]
Hi @tuxtechno 

Could you try by manually defining the level using the `-l` or `--level` param?
E.g. `-l n`

**Comment by tuxtechno** [comment]
I have checked config.py it is having min_level="none"
also tried specifying -l n still same issue find details below.

sudo python3 -m openvasreporting -i openvas.xml -f xlsx -l n

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/yogesh/openvasreporting/openvasreporting/__main__.py", line 19, in <module>
    main()
  File "/home/yogesh/openvasreporting/openvasreporting/openvasreporting.py", line 70, in main
    config = Config(args.input_files, 
  File "/home/yogesh/openvasreporting/openvasreporting/libs/config.py", line 86, in __init__
    raise ValueError("Invalid value for level parameter, \
ValueError: Invalid value for level parameter,                 must be one of: c[ritical], h[igh], m[edium], l[low], n[one]

**Comment by dudacgf** [comment]
I've pull requested a fix now, sorry about that

**Comment by TheGroundZero** [comment]
Fix should soon arrive in v1.5.2

**Comment by tuxtechno** [comment]
> Fix should soon arrive in v1.5.2

I really appreciate the support,  thanks it resolved the issue 👍 

I have few questions

1) how can i get host wise reporting in excel
2) is there any word / excel paid / free templates available to beautify the report.

**Comment by TheGroundZero** [comment]
> > Fix should soon arrive in v1.5.2
> 
> I really appreciate the support, thanks it resolved the issue 👍

You're welcome.
But @dudacgf is probably the one to thank the most.

> I have few questions
> 
>     1. how can i get host wise reporting in excel
> 
>     2. is there any word / excel paid / free templates available to beautify the report.

1. https://openvas-reporting.sequr.be/en/latest/usage/export-excel-host.html
2. I don't have any, feel free to make your own templates (https://github.com/TheGroundZero/openvasreporting/tree/master/openvasreporting/src)
